### PR TITLE
Fixing incorrect operator description

### PR DESCRIPTION
**Pull request by clockback**
This seems like an obvious mistake.

### CODE REVIEW

**Comment by hharrison** [comment]
Thanks so much for your patch, my apologies for not noticing sooner. Can you confirm you want credit simply as clockback in the commit message? Just a few stylistic things in case you decide to submit further changes to jaamsim:

We generally prefer our commits to contain a Signed-off-by: <Name> <email> and for contributers to acknowledge the contribution conforms to the following:

https://developercertificate.org/

Basically, that it is your work, or under a license compatible with jaamsim (in this case, the Apache 2.0 license)

Feel free to email me directly at harvey.harrison@gmail.com if you wish to discuss in private.

**Comment by clockback** [comment]
No thanks. I don't need accreditation, unless it is necessary, given that it is such a minor edit.

**Comment by hharrison** [comment]
OK, will get this folded in to the next release, thanks again.
